src/components/TodoList/TodoFilter.tsx

```diff
@@ -0,0 +1,66 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  filterTodo: (value: FilterType) => void,
+  selectedTodoFilter: FilterType,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  filterTodo, selectedTodoFilter,
+}) => {
+  const [selected, setSelected] = useState(true);
+
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        href="#/"
+        className={cn(
+          'filter__link', {
+            selected,
+          },
+        )}
+        data-cy="FilterLinkAll"
+        onClick={() => {
+          filterTodo(FilterType.ALL);
+          setSelected(true);
+        }}
+      >
+        All
+      </a>
+
+      <a
+        href="#/active"
+        className={cn(
+          'filter__link', {
+            selected: FilterType.ACTIVE === selectedTodoFilter,
+          },
+        )}
+        data-cy="FilterLinkActive"
```

Comment:
not critical but these elements looks very similar, i think will be better iterate this using `.map()`

Reply:
they differ on classNames and onClick action, so I'd prefer to leave them the way it is. Would help me to get back to this code and recall it;)